### PR TITLE
Add metainfo xml

### PR DESCRIPTION
**Pull request by nobrakal**
Add appdata files for plugins (like this used currently in fedora repo).

Great to see you add appdata files :D

### CODE REVIEW

File: data/vst-amsynth-plugin.metainfo.xml.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>vst-amsynth-plugin</id>
+  <extends>amsynth.desktop</extends>
+  <_name>VST plugin</_name>
+  <_summary>Amsynth plugin for the vst protocl</_summary>
```
Comment (koernepr) [inline data/vst-amsynth-plugin.metainfo.xml.in]: "protocl" -> "protocol"

**Comment by koernepr** [comment]
Some quick notes:

* Since the new files contain translatable entries, they need to be added to `po/POTFILES.in`; `make (dist)check` [fails](https://travis-ci.org/amsynth/amsynth/builds/212064466) for this reason.
* There's a small typo in `data/vst-amsynth-plugin.metainfo.xml.in` (see the note).
* Maybe you could capitalize "DSSI", "LV2" and "VST" both in the `_name` and `_summary` elements for consistency reasons.

**Comment by nixxcode** [comment]
`po/POTFILES.in` definitely needs updating to fix the Travis CI build failure;

```
$ make V=1 check
Making check in po
make[1]: Entering directory `/home/travis/build/amsynth/amsynth/po'
file=`echo de | sed 's,.*/,,'`.gmo \
	  && rm -f $file && /usr/bin/msgfmt -o $file de.po
file=`echo fr | sed 's,.*/,,'`.gmo \
	  && rm -f $file && /usr/bin/msgfmt -o $file fr.po
INTLTOOL_EXTRACT="/usr/bin/intltool-extract" XGETTEXT="/usr/bin/xgettext" srcdir=. /usr/bin/intltool-update --gettext-package amsynth --pot
rm -f missing notexist
srcdir=. /usr/bin/intltool-update -m
The following files contain translations and are currently not in use. Please
consider adding these to the POTFILES.in file, located in the po/ directory.
data/dssi-amsynth-plugin.metainfo.xml.in
data/lv2-amsynth-plugin.metainfo.xml.in
data/vst-amsynth-plugin.metainfo.xml.in
If some of these files are left out on purpose then please add them to
POTFILES.skip instead of POTFILES.in. A file 'missing' containing this list
of left out files has been written in the current directory.
Please report to https://github.com/amsynth/amsynth/issues
if [ -r missing -o -r notexist ]; then \
	  exit 1; \
	fi
make[1]: *** [check] Error 1
make[1]: Leaving directory `/home/travis/build/amsynth/amsynth/po'
make: *** [check-recursive] Error 1
The command "make V=1 check" exited with 2.
```

Pull requests need to pass all build checks before being merged.

**Comment by nobrakal** [comment]
Corrected

**Pull request merged**